pisek/jobs/cache.py

```diff
@@ -136,6 +136,7 @@ def move_to_top(self, entry: CacheEntry):
 
     def export(self) -> None:
         """Export cache into a file."""
-        with open(CACHE_CONTENT_FILE, "wb") as f:
+        with open(CACHE_CONTENT_FILE + ".tmp", "wb") as f:
             pickle.dump(self.cache, f)
+        os.rename(CACHE_CONTENT_FILE + ".tmp", CACHE_CONTENT_FILE)
```

Comment:
The path could be extracted into a variable:

```suggestion
        temporary_path = CACHE_CONTENT_FILE + ".tmp"
        with open(temporary_path, "wb") as f:
            pickle.dump(self.cache, f)
        os.rename(temporary_path, CACHE_CONTENT_FILE)
```